examples/microarch.yml

```diff
@@ -0,0 +1,32 @@
+mock: true
+type: fx32.32
+ioSync: Sync
+puLibrary:
```

Comment:
Hmm. Here we have misunderstanding. The idea behind this: describe library of process unit to be allocated in separated place, with name "library" intead of "proto"